alacritty_terminal/src/tty/windows/child.rs

```diff
@@ -31,7 +32,11 @@ extern "system" fn child_exit_callback(ctx: *mut c_void, timed_out: BOOLEAN) {
     }
 
     let event_tx: Box<_> = unsafe { Box::from_raw(ctx as *mut ChildExitSender) };
-    let _ = event_tx.sender.send(ChildEvent::Exited);
+    let child_handle = event_tx.child_handle.load(Ordering::Relaxed) as HANDLE;
+    let mut exit_code = 0_u32;
```

Comment:
Could you use the actual type of this value? `LPDWORD` or however windows-sys calls it.

Reply:
There seems to be no `LPDWORD` or even `DWORD` type in the `windows_sys` crate, and looks like tjeu use `*mut u32` directly:

* https://docs.rs/windows-sys/latest/windows_sys/Win32/System/Threading/fn.GetExitCodeProcess.html

* https://github.com/microsoft/windows-rs/blob/1f40da8ffc44adba3f9a866287108e80223d0a81/crates/libs/sys/src/Windows/Win32/System/Threading/mod.rs#L140

* https://github.com/microsoft/windows-rs/blob/1f40da8ffc44adba3f9a866287108e80223d0a81/crates/libs/windows/src/Windows/Win32/System/Threading/mod.rs#L840

Reply:
then it's fine.